simpleimage/make_simpleimage.sh

```diff
@@ -17,12 +17,23 @@
 
 set -e
 
-out="$1"
-disk_size="$2"
-kernel_tarball="$3"
+type="$1"
+out="$2"
+disk_size="$3"
+kernel_tarball="$4"
 
 if [ -z "$out" ]; then
-	echo "Usage: $0 <image-file.img> [disk size in MiB] [<kernel-tarball>]"
+	echo "Usage: $0 <image-type> <image-file.img> [disk size in MiB] [kernel-tarball]"
```

Comment:
Please do not change the usage of existing scripts. I want to keep it backwards compatible, so add new parametes to the end.